### PR TITLE
Fix machinepool template links

### PR DESCRIPTION
**Pull request by benjaminguttmann-avtq**
**What type of PR is this?**

/documentation

**What this PR does / why we need it**:

Fixes some broken links within the machinepool book topic 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

**Special notes for your reviewer**:

**Checklist**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests

**Release note**:

```release-note

Fixes broken links to templates in the machinepool book topic

```

### CODE REVIEW

**Comment by benjaminguttmann-avtq** [comment]
... so the CLA is signed in both Linux Foundation Platforms.. in the newer I added the github org as well... not sure what else I need to do ... 

/check-cla

**Comment by benjaminguttmann-avtq** [comment]
I signed it

File: docs/book/src/topics/machinepools.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ clusterctl config cluster my-cluster --kubernetes-version v1.16.8 --flavor machi
 ```
 
 The template used for this [flavor](https://cluster-api.sigs.k8s.io/clusterctl/commands/config-cluster.html#flavors)
-is located [here](../../templates/cluster-template-machinepool.yaml).
+is located [here](../../../../templates/cluster-template-machinepool.yaml).
```
Comment (randomvariable) [inline docs/book/src/topics/machinepools.md:28]:
```suggestion
is located [here](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/templates/cluster-template-machinepool.yaml).
```
Only occurred to me now, but given that this page is published at https://cluster-api-aws.sigs.k8s.io/templates/cluster-template-machinepool.yaml , should be a full URL back to Github.

File: docs/book/src/topics/machinepools.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ clusterctl config cluster my-cluster --kubernetes-version v1.16.8 --flavor eks-m
 ```
 
 The template used for this [flavor](https://cluster-api.sigs.k8s.io/clusterctl/commands/config-cluster.html#flavors)
-is located [here](../../templates/cluster-template-eks-managedmachinepool.yaml).
+is located [here](../../../../templates/cluster-template-eks-managedmachinepool.yaml).
```
Comment (randomvariable) [inline docs/book/src/topics/machinepools.md:51]:
```suggestion
is located [here](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/templates/cluster-template-eks-managedmachinepool.yaml).
```

Same as above.

**Review by randomvariable — CHANGES_REQUESTED** [review]

**Comment by benjaminguttmann-avtq** [comment]
@randomvariable ready for re-review

**Review by randomvariable — APPROVED** [review]

**Comment by randomvariable** [comment]
/lgtm
/approve

Thanks for spotting the error, and battling the CLA demons!

**Pull request merged**